3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,161 @@
+
+//Flight Booking
+function getFullname(firstname, surname, useFormalName = false, gender = "male") {
+    if (firstname === "" || surname === "") {
+        return "";
+    }
+
+    if (useFormalName) {
+        if (gender === "male") {
+            return "Lord " + firstname + " " + surname;
+        } else if (gender === "female") {
+            return "Lady " + firstname + " " + surname;
+        }
+    } else {
+        return firstname + " " + surname;
+    }
+}
+
+// Test cases
+const fullname1 = getFullname("Benjamin", "Hughes", true, "male");
+const fullname2 = getFullname("Halima", "Hassan", true, "female");
+
+console.log(fullname1);
+console.log(fullname2);
+
+
+// Event Application
+function getEventWeekday(daysFromToday) {
+    // Getting today's date
+    const today = new Date();
+
+
+    const futureDate = new Date(today.getTime() + daysFromToday * 24 * 60 * 60 * 1000);
+
+    const weekdays = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    const eventWeekday = weekdays[futureDate.getDay()];
+
+    return eventWeekday;
```

Comment:
The choice of variable names enhances the readability of your code. Well done! 👏🏻